README.md

```diff
@@ -8,15 +8,15 @@ Tokens will be validated for:
 * Expiration
 * Signature
 * Valid issuer
-* Valid Client/Audience
+* Valid Audience
 
 Basic usage:
 
 ``` java
 // 1. build the parser
 JwtVerifier jwtVerifier = new JwtHelper()
                             .setIssuerUrl("https://dev-123456.oktapreview.com/oauth2/ausar5cbq5TRooicu812")
-                            .setClientOrAudience("my-audience")
+                            .setAudience("my-audience")
```

Comment:
Also, re: `set` vs. `for`, for java set/get is the standard and is used by other libs to introspect and set values automatically.  I've seen other fluent libs include additional `withFoo` methods, but keeping consistent with our other java libs, our setters just return the current object so they can be chained together.